src/servers/src/error.rs

```diff
@@ -752,15 +766,13 @@ impl IntoResponse for Error {
             | Error::InvalidPromRemoteRequest { .. }
             | Error::InvalidQuery { .. }
             | Error::TimePrecision { .. } => HttpStatusCode::BAD_REQUEST,
```

Comment:
Do we need to log a trace log in this case? cc @sunng87

Reply:
I think it's ok to ignore them from log because we will return the error message to user for this type. For internal errors, we only give user a error code and `Internal Error` message, so it's necessary to log detail by our own.